Source/FullScreenPokemon.ts

```diff
@@ -4829,6 +4852,40 @@ module FullScreenPokemon {
                 callback);
         }
 
+        /* partyActivate functions
+         */
+
+        /**
+         * Makes sure that Player is facing the correct HMSolid
+         *
+         * @param player   The Player.
+         * @param pokemon   The Pokemon using the move.
+         * @param move   The move being used.
+         * @todo Eventually add check to make sure the Player beat the Gym leader needed to use the move.
+         * @todo Add context for what happens if player is not bordering the correct HMSolid.
+         */
+        partyActivateCheckThing(player: IPlayer, pokemon: IPokemon, move: IMoveSchema): void {
+            var borderedThing: IThing = player.bordering[player.direction];
+
+            if (borderedThing && borderedThing.title === move.solidName) {
+                move.partyActivate(player, pokemon);
+            }
+        }
+
+        /**
+         * Cuts a CuttableTree.
+         *
+         * @param player   The Player.
+         * @param pokemon   The Pokemon using Cut.
+         * @todo Eventually add check to make sure the Player beat the Gym leader needed to use the move.
+         * @todo Add an animation for what happens when the CuttableTree is cut.
+         */
+        partyActivateCut(player: IPlayer, pokemon: IPokemon): void {
+            player.FSP.MenuGrapher.registerB();
+            player.FSP.MenuGrapher.registerB();
```

Comment:
Oh, did not realize MenuGraphr didn't have a `.deleteAllMenus()`. I'll add that in after this.